opentracing/src/main/java/opentracing/TraceCodec.java

```diff
@@ -24,16 +24,16 @@
  * binary message format, so that it can be inspected efficiently by the middleware / routing layers
  * without parsing the whole message.
  */
-public interface TraceContextCodec {
+public interface TraceCodec {
```

Comment:
>  If we assume we change the name to SpanContext, why not call this SpanContextCodec?

See the Rationalie in the commit msg: https://github.com/opentracing/opentracing-java/commit/02d5d15bb51866afc7b8ed9d035798032c0b0848